fairytale.md

```diff
@@ -1,2 +1,30 @@
 once upon a time
-there was a **beautiful** _princess_
+there was an **incredible** _dragon_
+
+```
+                         ^                       ^
+                         |\   \        /        /|
+                        /  \  |\__  __/|       /  \
+                       / /\ \ \ _ \/ _ /      /    \
+                      / / /\ \ {*}\/{*}      /  / \ \
+                      | | | \ \( (00) )     /  // |\ \
+                      | | | |\ \(V""V)\    /  / | || \| 
+                      | | | | \ |^--^| \  /  / || || || 
+thoughts fly         / / /  | |( WWWW__ \/  /| || || ||
+upon the wings      | | | | | |  \______\  / / || || || 
+of fantasy          | | | / | | )|______\ ) | / | || ||
```

Comment:
This text/lyrics feels out or place without explanation for the reader.

Reply:
I decided to keep the text and add an introduction as you suggested, not to tamper the original artwork.